openreview/stages/venue_stages.py

```diff
@@ -1155,6 +1163,39 @@ def get_mandatory_readers(self, conference, number):
         if conference.use_senior_area_chairs:
             readers.append(conference.get_senior_area_chairs_id(number))
         return readers
+    
+    def get_description(self, conference):
+
+        if self.description:
+            return self.description
+        
+        instructions = '''
+- Please provide your official comments in the space provided below.
+- You can select who can read your comments.
+- You can also reply to existing comments.'''
+
+        if self.email_pcs:
+            instructions += '''
+- Program Chairs will be notified of your comments.'''
+        elif self.email_pcs_for_direct_comments:
+            instructions += '''
+- Program Chairs will be notified of your direct comments, only mandatory readers.'''
+        else:
+            instructions += '''
+- Program Chairs will not be notified of your comments.'''
+
+        if conference.use_senior_area_chairs:
+            if self.email_sacs:
+                instructions += '''
+- Senior Area Chairs will be notified of your comments.'''
+            else:
```

Comment:
It looks like if PCs select no to sending emails to SACs in the request form, this really means that they will be notified if the comment is visible only to mandatory readers. Maybe we should make this more clear in the venue request?

Reply:
that's true, the current option is "No, do not email SACs for each official comment made in the venue". The new options can say "No, do not email SACs for each official comment. Only send email notifications for comments that are specifically visible to PCs and SACs."

What do you think?

Reply:
I would say something like: 

> No, do not email SACs for each official comment; instead send email notifications for comments that are only visible to PCs and SACs.

We could also word it similar to how we word it for the PCs:

'Yes, email SACs for each official comment.',
'Yes, email SACs only for private official comments made in the venue (comments visible only to Program Chairs and Senior Area Chairs)',

Reply:
Minor suggestion, but I think it's strange to ask PCs `"Should the SACs(if applicable) receive an email for each official comment made in the venue?"` and both options say Yes, even though it functions the same as before.

What do you think about removing "Yes" and just writing:
- Email SACs for each official comment made in the venue
- Email SACs only for private official comments made in the venue (comments visible only to Program Chairs and Senior Area Chairs)

Or we could change the description.

Reply:
I like that too but if I change the first option (the most used), I would need to keep backward compatibility with all the existing venues. I wouldn't want to do that now, I would prefer to use your suggestion in the new UI.